### PR TITLE
Removing user field requirements (Issue 126)

### LINKED ISSUE #126: Simplificação do cadastro Mudamos Web
Tornar obrigatório só os dados de Nome, E-mail e Senha. Os demais dados seguem como dados de perfil do usuário, mas não obrigatórios. Ele pode entrar com os dados mínimos e completar o perfil em outro momento.

### PR DESCRIPTION
**Pull request by erickzanardo**
This PR closes #126

### How was it before?

- the had to input a lot of information upon registration, like birthday date, gender and etc
- the profile edition page was with error

### What has changed?

- the requirements was removed, leaving only name, email and password required for the registration
- the profile edition page was fixed

### What should I pay attention when reviewing this PR?

Nothing special

### Is this PR dangerous?

no

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -342,8 +342,6 @@ def self.to_csv
   validates_attachment :picture, content_type: { content_type: ["image/jpeg", "image/gif", "image/png", "image/jpg"] }, unless: -> { self.first_step }
 
   validates_presence_of :name
-  validates_presence_of :birthday, unless: -> { self.is_admin }
```
Comment (guimello) [inline app/models/user.rb:345]: Are there any impact on the app by removing this requirement? I mean, is there any code that expects this to be set?

**Review by guimello — COMMENTED** [review]
Won't you be changing the forms to make it easier to the user or something like that?

File: app/models/user.rb (hunk 1 above)
Comment (erickzanardo) [reply app/models/user.rb:345]: I don't think so, but I will check

**Comment by erickzanardo** [comment]
@guimello actually we have another issue for that, this PR is only meant to remove fields requirements.

File: app/models/user.rb (hunk 1 above)
Comment (erickzanardo) [reply app/models/user.rb:345]: Actually there are some impact, but this will be handled on another issue/PR

**Review by guimello — APPROVED** [review]

**Pull request merged**